lib/foreman_ansible/engine.rb

```diff
@@ -29,6 +29,12 @@ class Engine < ::Rails::Engine
 
     initializer 'foreman_ansible.register_plugin', :before => :finisher_hook do
       require 'foreman_ansible/register'
+
+      Foreman::Plugin.register :foreman_ansible do
```

Comment:
this is already in `lib/foreman_ansible/register.rb` file, we should not add another registration block, it could do bonker stuff.